seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/utils/JdbcCatalogUtils.java

```diff
@@ -63,13 +64,17 @@ public class JdbcCatalogUtils {
     private static final String DEFAULT_CATALOG_NAME = "jdbc_catalog";
 
     public static Map<TablePath, JdbcSourceTable> getTables(
-            JdbcConnectionConfig jdbcConnectionConfig, List<JdbcSourceTableConfig> tablesConfig)
+            JdbcConnectionConfig jdbcConnectionConfig,
+            List<JdbcSourceTableConfig> tablesConfig,
+            JdbcSourceConfig sourceConfig)
```

Comment:
```
JdbcSourceConfig sourceConfig
```
change to
```
String dialectName
```
since we only use this parameter.

Reply:
will update